Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -107,6 +117,36 @@ private static void CreateInputActionReferences(InputActionAsset asset)
             }
         }
 
+        internal static void CheckForDefaultUIActionMapChanges()
+        {
+            var asset = GetOrCreate();
+            if (asset != null)
+            {
+                if (s_DefaultUIMap == null)
+                    GetDefaultUIActionMap();
+                
+                var uiMap = asset.actionMaps.FirstOrDefault(m => m.name == "UI");
```

Comment:
LINQ usage.

Reply:
I'll leave them for now since it only impacts the Editor and this class already has LINQ references.